<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/img/img/note.png</normaloff>:/res/img/img/note.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="CodeEditor" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menu_F_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionWordWrap"/>
    <addaction name="actionLineNum"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionFontColor"/>
    <addaction name="separator"/>
    <addaction name="actionFontBackColor"/>
    <addaction name="actionBackColor"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionToolbar"/>
    <addaction name="actionStatusbar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_F_2"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionWordWrap"/>
   <addaction name="actionLineNum"/>
   <addaction name="actionToolbar"/>
   <addaction name="actionStatusbar"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/new.png</normaloff>:/res/img/img/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/open.png</normaloff>:/res/img/img/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/save2.png</normaloff>:/res/img/img/save2.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/saveas.png</normaloff>:/res/img/img/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/exit.png</normaloff>:/res/img/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/recall.png</normaloff>:/res/img/img/recall.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/cut.png</normaloff>:/res/img/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/copy.png</normaloff>:/res/img/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/paste2.png</normaloff>:/res/img/img/paste2.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/seek.png</normaloff>:/res/img/img/seek.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/replace3.png</normaloff>:/res/img/img/replace3.png</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/font.png</normaloff>:/res/img/img/font.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;Q)</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/fontcolor.png</normaloff>:/res/img/img/fontcolor.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/all.png</normaloff>:/res/img/img/all.png</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/linefeed.png</normaloff>:/res/img/img/linefeed.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="actionLineNum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/num.png</normaloff>:/res/img/img/num.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>显示行号(L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/renew.png</normaloff>:/res/img/img/renew.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/tool.png</normaloff>:/res/img/img/tool.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(&amp;T)</string>
   </property>
  </action>
  <action name="actionStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/state.png</normaloff>:/res/img/img/state.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(&amp;B)</string>
   </property>
  </action>
  <action name="actionFontBackColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/fontback.png</normaloff>:/res/img/img/fontback.png</iconset>
   </property>
   <property name="text">
    <string>字体背景色</string>
   </property>
  </action>
  <action name="actionBackColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/back.png</normaloff>:/res/img/img/back.png</iconset>
   </property>
   <property name="text">
    <string>编辑器背景色</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/img/img/about.png</normaloff>:/res/img/img/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;U)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
